React/DevSupport/RCTPackagerClient.h

```diff
@@ -25,7 +25,7 @@ extern const int RCT_PACKAGER_CLIENT_PROTOCOL_VERSION;
 @protocol RCTPackagerClientMethod <NSObject>
 
 - (void)handleRequest:(NSDictionary<NSString *, id> *)params withResponder:(RCTPackagerClientResponder *)responder;
-- (void)handleNotification:(NSDictionary<NSString *, id> *)params;
+- (void)handlePackageNotification:(NSDictionary<NSString *, id> *)params;
```

Comment:
Shouldn't this be:

```suggestion
- (void)handlePackagerNotification:(NSDictionary<NSString *, id> *)params;
```

Is this change necessary to fix the uploading issue? It doesn't look like it would clash with anything. Apple's naming convention is usually something like `clientDidReceiveNotification:`. Since it's in a header file, it's kind of public.

Reply:
This is mentioned in the [original](https://github.com/facebook/react-native/issues/31507#issue-885801684) issue and is a part of the error message: 
> ITMS-90338: Non-public API usage - The app references non-public selectors in app: _isKeyDown, _modifiedInput, _modifierFlags, handleNotification:, isPassthrough, onSuccess:, removeValuesForKeys:completion:, setCategoryID:. If method names in your source code match the private Apple APIs listed above, altering your method names will help prevent this app from being flagged in future submissions. In addition, note that one or more of the above APIs may be located in a static library that was included with your app. If so, they must be removed. For further information, visit the Technical Support Information at http://developer.apple.com/support/technical/

So it probably is a problem?